java/src/client/game/GameManager.java

```diff
@@ -54,6 +54,7 @@
      * @post This provides valid operations on GameManager
      */
     GameManager() {
+        clientModel = new ClientModel();
```

Comment:
yeah there is prob a better way, but i don't want to break everyones stuff by making it null